index.js

```diff
@@ -16,12 +16,40 @@ mongoose
   })
   .then(self => {
     console.log(`Connected to the database: "${self.connection.name}"`);
-    // Before adding any recipes to the database, let's remove all existing ones
+    // Before adding any recipes to the database, let's remove all existing ones);
     return Recipe.deleteMany()
+  }).then(() => Recipe.syncIndexes())
+    .then( () =>  Recipe.create(
+    {
+    title: "Milanesas con Papas Fritas",
+    level:'Amateur Chef',
+    ingredients: [
+      '8 beef steaks 1cm thick',
+      '1 bag of breadcrumbs', 
+      '6 Eggs',
+      '1 pinch cayenne pepper',
+      '1 pinch salt'
+    ],
+    cuisine:'Argentina',
+    dishType: 'main_course',
+    image: 'https://static3.diariovasco.com/www/multimedia/202107/16/media/cortadas/MILANESA-R7jYwhgmCWZSZc1g1kG5qzM-624x385@Diario%20Vasco.jpg',
+    duration: 60,
+    creator: "Chef Manuel",
+    created: '',
+  }))
+  .then(recipe => console.log('Se ha creado estos registros:', Recipe))
```

Comment:
- Aquí, el `console.log` te está mostrando el modelo `Recipe`, si lo que querías era mostrar por consola la receta que acabas de crear, puedes poner lo siguiente (`recipe` en minúscula, porque se refiere a lo que devuelve la promesa)

```suggestion
  .then(recipe => console.log('Se ha creado este registro:', recipe))
```